pkg/util/shard.go

```diff
@@ -3,6 +3,13 @@ package util
 import (
 	"crypto/md5"
 	"encoding/binary"
+	"math"
+)
+
+const (
+	// Sharding strategies.
+	ShardingStrategyDefault = "default"
+	ShardingStrategyShuffle = "shuffle-sharding"
```

Comment:
I moved constants here to be able to reference them in ingester package without having to import distributor. I took the opportunity to use these constants in the store-gateway too.

Reply:
Same constants are used by ruler now.

Reply:
Good point, done!